### PR TITLE
Implement missing opcodes "Is in area?"

### PR DESCRIPTION
**Pull request by ShFil119**
Small pull, it's practically copy and paste.
Edit. I've added one more bigger commit. It's still calque, but I'm not sure about int arg6/8.
And how think about angled area.

### CODE REVIEW

File: rwengine/src/script/modules/GTA3ModuleImpl.inl [hunk 1]
```diff
@@ -4792,13 +4712,12 @@ bool opcode_019f(const ScriptArguments& args, const ScriptPlayer player, const S
 	opcode 01a0
 	@arg player 
 	@arg coord0
-	@arg coord1
+	@arg coord1
```
Comment (JayFoxRox) [inline rwengine/src/script/modules/GTA3ModuleImpl.inl]: unintended?

File: rwengine/src/script/modules/GTA3ModuleImpl.inl [hunk 2]
```diff
@@ -4792,13 +4712,12 @@ bool opcode_019f(const ScriptArguments& args, const ScriptPlayer player, const S
...
 	@arg coord0
-	@arg coord1
+	@arg coord1 
 	@arg arg8 
 */
 bool opcode_01a0(const ScriptArguments& args, const ScriptPlayer player, const ScriptVec3 coord0, const ScriptVec3 coord1, const ScriptInt arg8) {
-	auto plyChar = player->getCharacter();
-	auto condition = plyChar->isStopped() && plyChar->getCurrentVehicle() != nullptr;
-	return script::objectInArea(args, plyChar, coord0, coord1, arg8, condition);
+	auto condition = player->getCharacter()->isStopped() && player->getCharacter()->getCurrentVehicle() != nullptr;
```
Comment (JayFoxRox) [inline rwengine/src/script/modules/GTA3ModuleImpl.inl]: probably makes more sense to first check the vehicle state.
The `isStopped` might not even be necessary and probably depends on knowing if the char or their vehicle has stopped. You should also check what the precise condition for a vehicle having stopped is (forward / backward speed close to zero? Overall speed close to zero? Characters speed close to zero? Handbrake applied?).

If we can't figure it out now, we should at least keep an eye on those details (and track them).

Comment (ShFil119) [reply rwengine/src/script/modules/GTA3ModuleImpl.inl]: As far I know function update will set flag stopped when speed is close to zero. I'll add extra condition checking vehicle.

File: rwengine/src/script/modules/GTA3ModuleImpl.inl (hunk 1 above)
Comment (ShFil119) [reply rwengine/src/script/modules/GTA3ModuleImpl.inl]: Pardon, file should be reformatted. Some lines are ended with space, some not.

**Comment by ShFil119** [comment]
Updated:
Used JaxFoxRox's suggestions.
Converted arguments to be constant.
Fix : some of opcodes should use objectInRadius.

**Pull request merged**